doc/nrf/releases_and_maturity/migration/migration_hwmv2.rst

```diff
@@ -50,21 +50,34 @@ To convert your board from hardware model v1 to v2, consult the following pages:
 * The `ncs-example-application conversion commit <ncs-example-application commit f9f2da_>`_ as an example of how to port a simple board.
 * The `hardware model v2 commit <Zephyr commit 8dc3f8_>`_, containing the full conversion of all existing boards from the old to the current model, to use as a complete conversion reference.
 
+.. note::
+   Board names must be unique, so you cannot boards in hardware model v1 and v2 description using the same name.
+   If you decide to create a new board in hardware model v2 instead of converting an existing board then ensure that corresponding board config fragments and devicetree overlays for the new boards are also created and that any tests for the board in Kconfig, build system, and source code are correctly updated.
```

Comment:
```suggestion
   If you decide to create a new board in hardware model v2 instead of converting an existing board, ensure that corresponding board config fragments and devicetree overlays for the new boards are also created and any tests for the board in Kconfig, build system, and source code are correctly updated.
```